internal/wrapnix/wrapper.sh.tmpl

```diff
@@ -25,12 +25,13 @@ eval "$(DO_NOT_TRACK=1 devbox shellenv --preserve-path-stack -c {{ .ProjectDir }
 fi
 
 {{/*
-We call only-path-without-wrappers so that we do not invoke other bin-wrappers from
+Remove wrapper bin path from PATH so that we don't call more bin-wrappers from
 this bin-wrapper. Instead, we directly invoke the binary from the nix store, which
 should be in PATH.
 
-DO_NOT_TRACK=1 can be removed once we optimize segment to queue events.
+This is implemented in sed for efficiency. sed is POSIX so we assume it's available.
```

Comment:
oh interesting. Availability was one of the concerns earlier.

Reply:
@savil if it's a concern we can implement in pure bash. Here's an implementation that only uses bash builtins:

```bash
IFS=: read -ra dirs <<< "$PATH"  # Split PATH into an array using colon as a delimiter

new_PATH=""
for dir in "${dirs[@]}"; do
    if [[ "$dir" != "/path/to/remove" ]]; then
        if [[ -z "$new_PATH" ]]; then
            new_PATH="$dir"
        else
            new_PATH="$new_PATH:$dir"
        fi
    fi
done

export PATH="$new_PATH"
```

Reply:
oh not too bad. That's fairly understandable except for:
```
IFS=: read -ra dirs <<< "$PATH" 
```

:D

Reply:
but now that makes sense too...